src/client/components/filter-tile/filter-tile.tsx

```diff
@@ -417,12 +417,20 @@ export class FilterTile extends React.Component<FilterTileProps, FilterTileState
     </div>;
   }
 
+  renderTimeShiftLabel(dimension: Dimension): string {
+    const { essence } = this.props;
+    if (!dimension.expression.equals(essence.dataCube.timeAttribute)) return null;
+    if (!essence.hasComparison()) return null;
+    return `(Shift: ${essence.timeShift.valueOf().getDescription(true)})`;
```

Comment:
Could you encapsulate essence.timeShift.valueOf().getDescription(true) somehow? E.g essence.timeShift.getDescription()